webpack.config.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+const path = require('path')
+
+const webpack = require('webpack')
+
+// Set this to false for easier debugging:
+// TODO: Production mode is currently crashing
+const production = false
+
+const babelOptions = {
+  presets: production ? ['@babel/preset-env'] : [],
```

Comment:
Actually, wait! This will completely break on old Android phones. I need to solve this before we can merge anything.

Reply:
Aha! Found the problem, and we now run with minified plugins. The plugin bundle is now 4.36 MiB.